README.md

```diff
@@ -65,55 +65,65 @@ with:
         redeemTask: true
       dataPurge:
       formattingEvaluator:
-        scores:
-          br: 0
-          code: 1
-          p: 1
-          em: 0
-          img: 0
-          strong: 0
-          blockquote: 0
-          h1: 1
-          h2: 1
-          h3: 1
-          h4: 1
-          h5: 1
-          h6: 1
-          a: 1
-          li: 1
-          td: 1
-          hr: 0
         multipliers:
           - select: [ ISSUE_SPECIFICATION ]
             formattingMultiplier: 1
-            wordValue: 0.1
+            symbols:
```

Comment:
I always associate symbols with special characters like ^%$#, so maybe a better name would be `regexes` or `regexMultiplier`?

Reply:
Named it after the spec, I can change it. @0x4007 What do you prefer?

Reply:
Not sure, perhaps `pattern: ` however I know I had another task somewhere for custom aliases to map to specific regexes so that its more readable in the config. Or we can consider hard coding these inside of the plugin. For example:

```yml
alias: 
   "\\b\\w+\\b": "words"
```

And then later in the config we can just do something like
```yml
pattern: 
   words: 0.1
```

I have a feeling hard coding the common useful ones might be best

Reply:
Sure we can have another task opened for that. Shall I rename to `pattern` in the meantime here?

Reply:
Let's do what was defined in my example config above. 

https://github.com/ubiquibot/conversation-rewards/pull/68#discussion_r1696576344